### PR TITLE
NIST NVD 2.0 - KEV causing feed to timeout.

### PR DESCRIPTION
**Pull request by Ni-Knight**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-37575

## Description
When KEV was requested by the user a malformed URL was created which caused a timeout. 
Fixed the formation of the URL.

### CODE REVIEW

File: Packs/FeedNVDv2/Integrations/FeedNVDv2/FeedNVDv2.py [hunk 1]
```diff
@@ -71,6 +71,7 @@ def build_param_string(self, params: dict) -> str:
 
         param_string: str = '&'.join([f'{key}={value}' for key, value in params.items()])
         param_string = param_string.replace('noRejected=None', 'noRejected')
+        param_string = param_string.replace('hasKev=True', 'hasKev')
```
Comment (bziser) [inline Packs/FeedNVDv2/Integrations/FeedNVDv2/FeedNVDv2.py:74]: looks good, please add to the test `test_build_param_string`  this usecase

File: Packs/FeedNVDv2/ReleaseNotes/1_0_2.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### National Vulnerability Database Feed v2
+
+- Fixed an issue that caused the integration to timeout when kev parameter was selected by the user.
```
Comment (ShirleyDenkberg) [inline Packs/FeedNVDv2/ReleaseNotes/1_0_2.md:6]:
```suggestion
Fixed an issue that caused the integration to timeout when the *kev* parameter was selected by the user.
```

**Comment by ShirleyDenkberg** [comment]
@bziser Doc review completed.

**Review by bziser — APPROVED** [review]
great work!

**Pull request merged**